exams/pearson.py

```diff
@@ -141,6 +149,19 @@ def _profile_country_to_alpha3(profile):
 ], field_prefix='profile')
 
 
+write_ead_file = _tsv_writer([
+    ('AuthorizationTransactionType', 'operation'),
+    ('ClientAuthorizationID', 'id'),
+    ('ClientCandidateID', 'user.profile.student_id'),
+    ('ExamSeriesCode', 'course.program.exam_series_code'),
+    ('Modules', 'course.exam_module'),
+    ('Accommodations', lambda _: ''),
+    ('EligibilityApptDateFirst', lambda exam_auth: _format_date(exam_auth.date_first_eligible)),
+    ('EligibilityApptDateLast', lambda exam_auth: _format_date(exam_auth.date_last_eligible)),
+    ('LastUpdate', lambda exam_auth: _format_datetime(exam_auth.updated_on)),
+])
```

Comment:
I'm probably going to go in the direction of converting these to classes instead because it's going to be easier to mock and reason about.